server/queue/queue-api/src/test/java/org/apache/james/queue/api/ManageableMailQueueContract.java

```diff
@@ -420,6 +424,22 @@ default void browseShouldNotFailWhenConcurrentRemoveWhenIterating() throws Excep
         assertThatCode(() ->  Iterators.consumingIterator(items)).doesNotThrowAnyException();
     }
 
+    @Test
+    default void browseShouldReturnMailsWithMimeMessage() throws Exception {
```

Comment:
I thought we decided exactly the opposite, did I missed something ?

Reply:
Browsing needs the mime-message (or the admin is blind) but some operations like updating browse start / removes do not need it.